packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -456,9 +458,9 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
     this._classNames = getClassNames(styles!, {
       legendColor: this.state.color,
       theme: theme!,
-      width: width,
+      width,
```

Comment:
revert all these changes

Reply:
ok. this was caused because of prettier/

Reply:
I removed this shorthands but whenever I am committing it is automatically adding shorthands.